async_timeout/__init__.py

```diff
@@ -102,7 +102,7 @@ def _cancel_task(self) -> None:
             self._cancelled = True
 
 
-def current_task(loop: asyncio.AbstractEventLoop) -> asyncio.Task:
+def current_task(loop: asyncio.AbstractEventLoop) -> 'asyncio.Task[Any]':
```

Comment:
Why forward declaration here?
I agree to `Task` -> `Task[Any]` change but wonder why regular declaration doesn't work?

Reply:
`asyncio.Task` isn't generic at runtime, so this will throw a TypeError when the annotation is evaluated.

Reply:
Cool trick!